WordPress/Classes/Models/BasePost.h

```diff
@@ -45,8 +45,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 
 //date conversion
-- (nullable NSDate *)dateCreated;
-- (void)setDateCreated:(nullable NSDate *)localDate;
+@property (nonatomic, strong, nullable) NSDate * dateCreated;
```

Comment:
Haven't been able to repro any kind of glitch related to this one. I'll keep (both) eyes open, thanks for the heads up!!